src/app/shared/pipes/default-null-or-empty/default-null-or-empty.pipe.ts

```diff
@@ -8,7 +8,13 @@ export class DefaultNullOrEmpty implements PipeTransform {
 
   transform(value: any): any {
     if (typeof value === 'string') {
-      return value.trim().length > 0 ? this.titleCaseFirstWord(value) : '-';
+      if (value.toLowerCase() === 'true') {
```

Comment:
Worth unit testing this?

Reply:
Bruce wanted this asap. Our newly introduced custom components lack some unit tests too. We will need to have a coverage specific PR to address all those, like the one we did in the previous sprint.